### ISSUE TITLE
Explorer Patcher does not work correctly

### ISSUE THREAD

**Issue by dcimorse** [valinet/ExplorerPatcher#3006]
When I install Explorer Patcher it opens with a completely blank screen with no Taskbar. Nothing happens when I click on Desktop
Windows Start Key s a Start menu but it is not correctly formatted. 
Clicking on the screen does nothing. Cannot open Settings.

This situation occurred after I did a Windows update on Patch Tuesday

Is this fixable.
Thanks

P.S. There seems to be a Ghost copy of Explorer Patcher folder somewhere on my computer but I cannot delete it,

**Comment by Amrsatrio** [comment]
We need more information to assist with your problem:
1. Your Windows build number found in `winver`
2. The version of ExplorerPatcher you're using found in EP properties > About.

**Comment by weggetor** [comment]
Same issue here. 
=>Windows 11 23H2 (Build 22631.3296). 
=> latest release of explorer patcher : 22621.3296.64.1

**Comment by Amrsatrio** [comment]
@weggetor Please explain the behaviors you're expecting and the behaviors you're getting instead.

**Comment by weggetor** [comment]
Its very simple - as the original poster mentioned. Windows 11 Taskbar works fine. If I select Windows 10 Taskbar style and restart explorer, desktop and taskbar are all black with no icons

**Comment by Amrsatrio** [comment]
Looks like explorer.exe crashed. Do have any other customization software installed like Start11?

**Comment by weggetor** [comment]
No, nothing else

**Comment by Amrsatrio** [comment]
Please contact me on Discord to help me pin down the issue.

**Comment by Amrsatrio** [comment]
> True Launch Bar

Maybe? Try uninstalling them so EP remains to be as the only customization software.

**Comment by weggetor** [comment]
I'll try this - but not now. I'm still at work. But I do not think its related. It adds only a Toolbar to the windows 10 Taskbar

**Comment by dcimorse** [comment]
I have done some more investigating of the problem I presented in my OP. I have dual monitors and when EP opened and I did not see a taskbar I neglected to check the second monitor. On doing so I discovered there was a taskbar on that monitor but it was on the left side of the screen and it did not look correct.  Eventually I was able to open Properties and found that I had somehow set the taskbar location to the left - which caused it to be on the left side of the **second** monitor.
After resetting it to bottom and restarting EP, it returned to the bottom. I then had to close and reset my Desktop and Quick Launch toolbars.

Everything works as it should now. **_Thank you so much for developing this wonderful patch._** I didn't realize how much I depended on it for efficient day to day usage when I was forced to use plain vanilla W11 until I fixed the problem.
Keep up the good work
